rpcs3/Emu/Cell/lv2/sys_net/lv2_socket_native.cpp

```diff
@@ -143,6 +157,8 @@ s32 lv2_socket_native::bind(const sys_net_sockaddr& addr)
 		saddr = std::bit_cast<u32>(psa_in->sin_addr);
 	}
 
+	feign_tcp_conn_failure = false; // TODO
```

Comment:
I think that technically, reusing a socket from a dead connection is considered UB. Normal behaviour is just closing the socket and creating a new one(which games do). So it's hard to advise what would be the way to handle this case. I'd remove it and assume games don't do something insane lol.